api/pacman-api-statistics/src/main/java/com/tmobile/pacman/api/statistics/repository/StatisticsRepositoryImpl.java

```diff
@@ -168,6 +168,25 @@ public JsonArray getNumberOfAccounts() throws DataException {
 
     }
 
+    @Override
+    public JsonArray getNumberOfRedhatAccounts() throws DataException {
+        try {
+            JsonParser parser = new JsonParser();
+            StringBuilder urlToQueryBuffer = new StringBuilder(esUrl).append("/").append(REDHAT).append("/")
+                    .append(SEARCH);
+            StringBuilder requestBody = new StringBuilder(
+                    "{\"query\":{\"bool\":{\"must\":[{\"term\":{\"latest\":\"true\"}}]}},\"aggs\":{\"accounts\":{\"terms\":{\"field\":\"accountid.keyword\",\"size\":10000}}}}");

```

Comment:
does this include only RH accounts?

Reply:
yes, individual sum of redhat + aws + gcp +azure = total accounts under stats